src/app/services/courses.service.ts

```diff
@@ -21,19 +21,26 @@ export class CoursesService {
       .pipe(finalize(() => this.loaderService.hide()));
   }
 
+  getAuthors() {
+    this.loaderService.show();
+    return this.http.get('http://localhost:3004/authors')
```

Comment:
You can use generic method call signature here to tell Typescript what is the format of server response:
`return this.http.get<Author[]>('http://localhost:3004/authors')`

Reply:
done